documentation/usage.rst

```diff
@@ -0,0 +1,591 @@
+Sheetfu API usage
```

Comment:
Amazing documentation! Very clear to understand for newcomers, and even I learnt some new things I didn't know (like the editor thing in the sa.create() xD)

Thanks a lot!

Reply:
Thanks for the feedback. 
If you're fine with it, could you please merge it?

Reply:
Sure! If it's ok I'll do a "squash and merge" to avoid having all those "MAJ" commits in the commit-log =)

Reply:
Yeah, that's a good idea.